src/prefect/cli/get.py

```diff
@@ -80,7 +81,7 @@ def flows(name, version, project, limit, all_versions):
 
     headers = ["NAME", "VERSION", "AGE"]
 
-    if project:
+    if config.backend == "cloud" or project:
```

Comment:
What's the idea behind keeping the "or project" piece?

Reply:
The idea is that there is no idea